sdk/eventhub/azure-eventhub/tests/pyamqp_tests/unittest/test_websocket_exception.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+import asyncio
+from unittest.mock import patch, MagicMock
+
+import aiohttp
+from azure.eventhub._pyamqp.aio._transport_async import WebSocketTransportAsync
+
+
+# class WebsocketException(unittest.TestCase):
+async def test_websocket_aiohttp_exception():
```

Comment:
do we want to add a sync mock websocket test too?

Reply:
For websocket timeout exception? We can yup yup let me add it

Reply:
ah maybe not then

Reply:
We except on WebSocketTimeoutException which is thrown by websocket